test/extensions/filters/http/aws_request_signing/config_test.cc

```diff
@@ -93,7 +93,92 @@ host_rewrite: new-host
   cb(filter_callbacks);
 }
 
+TEST(AwsRequestSigningFilterConfigTest, SimpleConfigWithQueryString) {
+  const std::string yaml = R"EOF(
+service_name: s3
+region: us-west-2
+host_rewrite: new-host
+query_string: {}
+use_unsigned_payload: true
+match_excluded_headers:
+  - prefix: x-envoy
+  - exact: foo
+  - exact: bar
+  )EOF";
+
+  AwsRequestSigningProtoConfig proto_config;
+  TestUtility::loadFromYamlAndValidate(yaml, proto_config);
+
+  AwsRequestSigningProtoConfig expected_config;
+  expected_config.set_service_name("s3");
+  expected_config.set_region("us-west-2");
+  expected_config.set_host_rewrite("new-host");
+  expected_config.mutable_query_string();
```

Comment:
Do you think we can add this check?
`expected_config.mutable_query_string()->mutable_expiration_time()->set_seconds(5);`

Reply:
new test case added to verify that the default 5s expiration is being correctly set